README.md

```diff
@@ -1,88 +1,126 @@
+
+
+#### Prerequisites
+
+This project uses Apple's Swift programming language 3.0 for iOS, and the only supported way to develop iOS apps is on an Apple computer running OS X and Xcode. We have tested this application with the latest versions of iOS (10.0) and Xcode (8.0) at the time of this writing.
+
+
+#### About the Quickstart Project
 
 This project consists of two sample apps: one using CallKit, and one without CallKit support.
 
-The quick starts will help you get video chat integrated directly into your iOS applications using Twilio's Video SDK. These examples are for Swift developers - if your app uses Objective-C see [video-quickstart-objective-c](https://github.com/twilio/video-quickstart-objc/).
+The quickstarts will help you integrate Twilio Video directly into your iOS applications using Twilio's Video SDK. These examples are for Swift developers.
+
+**Note: if your app uses Objective-C see [video-quickstart-objective-c](https://github.com/twilio/video-quickstart-objc/).**
 
 The sample apps are written in Swift 3.0. You will need at least Xcode 8.0 in order to run the applications.
 
 Review the code to see how how to set up key classes like `TVIVideoClient`, `TVIRoom`, `TVIParticipant`, `TVILocalMedia`, and `TVICameraCapturer`. The ViewController implements the `TVIRoomDelegate`, and `TVIParticipantDelegate` protocols in order to display remote Participant video on screen.
 
-### Additional Examples
+Download this project and run `pod install` to install TwilioVideo.framework. Open VideoQuickStart.xcworkspace in Xcode.
 
-You will also find additional examples that provide more advanced use cases of the Video SDK. The currently included examples are as follows:
 
-- [Screen Capturer](ScreenCapturerExample) - Shows how to use `TVIScreenCapturer` to capture the contents of a `UIView`, and how a custom `TVIVideoCapturer` can be implemented to do the same.
 
-## Prerequisites
+## Quickstart
 
-This project uses Apple's Swift programming language 3.0 for iOS, and the only supported way to develop iOS apps is on an Apple computer running OS X and Xcode. We have tested this application with the latest versions of iOS (10.0) and Xcode (8.0) at the time of this writing.
 
-You can use the iOS Simulator that comes with Xcode to do your testing, but local video will not be shared since the Simulator cannot access a camera. If you have an iOS device, you can now run apps from Xcode on your device without a paid developer account.
 
-## Twilio Library Setup for the Project
+### Running the Quickstart
 
-You will need to add the Twilio Video library to the project to compile and run. You can do this using the [Cocoapods](https://cocoapods.org/) dependency management system. Version 1.0 or greater is required.
+To get started with the Quickstart application follow these steps:
 
-### Using Cocoapods
+1.Open this project in xCode and select the quickstart app module
```

Comment:
`Xcode` not `xCode`.

App module? I'm not sure what we are supposed to be doing in this step?